runtime/v2/logging/logging_unix.go

```diff
@@ -46,7 +46,10 @@ func Run(fn LoggerFunc) {
 	signal.Notify(sigCh, unix.SIGTERM)
 
 	go func() {
-		errCh <- fn(ctx, config, wait.Close)
+		errCh <- fn(ctx, config, func() error {
+			wait.Write([]byte{0})
```

Comment:
Why does it need the zero-bytes write?

/ok-to-test

Reply:
The intention was just "send something" so that the [reader end](https://github.com/containerd/containerd/blob/main/pkg/process/io.go#L299) could return. A single byte of "\0" was chosen randomly.